Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM kaazing/gateway:latest
```

Comment:
The docker image on:
https://registry.hub.docker.com/u/kaazing/gateway/dockerfile/ 
seems to not have Agrona included for now. Currently version 5.0.1.21 seems to be used. 

Available versions listed here: 
https://oss.sonatype.org/content/repositories/releases/org/kaazing/gateway.distribution/

Version 5.0.1.27 seems to have Agrona available. Can a switch to this version be made?